subworkflows/local/vcf_concatenate_bcftools/main.nf

```diff
@@ -11,8 +11,7 @@ workflow VCF_CONCATENATE_BCFTOOLS {
     ch_versions = Channel.empty()
 
     // Remove chromosome from meta
-    ch_vcf_tbi_grouped = ch_vcf_tbi
-        .map{ meta, vcf, tbi -> [['id' : meta.id], vcf, tbi] }
+    ch_vcf_tbi_grouped = ch_vcf_tbi.map{ meta, vcf, tbi -> [['id' : meta.id], vcf, tbi] }
 
     // Group by ID
     ch_vcf_tbi_grouped = ch_vcf_tbi_grouped.groupTuple( by:0 )
```

Comment:
I'm not sure if the grouping needs to be done inside this subworkflow or outside when we call it.
What do you think ?

Reply:
I prefer them inside subworkflows, looks tidier, but I am not sure if there is a recommendation/best-practice around this. Sarek also seems to do these type of channel manipulation inside the local subworkflows ([example here](https://github.com/nf-core/sarek/blob/ea88402912c329b4fd234ad07294ce05bbd2590c/subworkflows/local/bam_applybqsr/main.nf))